community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/RewindableExecutionResult.scala

```diff
@@ -76,7 +78,7 @@ object RewindableExecutionResult {
   private def compatibility(inner: v3_1.executionplan.InternalExecutionResult, planner: v3_1.PlannerName, runtime: v3_1.RuntimeName): InternalExecutionResult = {
     val result: v3_1.executionplan.InternalExecutionResult = inner match {
       case other: v3_1.PipeExecutionResult =>
-        exceptionHandlerFor2_3.runSafely {
+        exceptionHandlerFor3_1.runSafely {
```

Comment:
It is. It's test-only and I don't think we ever use the exception handling in the tests, so I don't think it's important to backport though.